src/dataLoaders/actions/dataLoaders.ts

```diff
@@ -376,43 +372,25 @@ export const createOrUpdateTelegrafConfigAsync = () => async (
   getState: GetState
 ) => {
   const {
-    telegrafPlugins,
     telegrafConfigID,
     telegrafConfigName,
     telegrafConfigDescription,
   } = getDataLoaders(getState())
-  const {name} = getOrg(getState())
-  const {bucket} = getSteps(getState())
-
-  const influxDB2Out = {
-    name: TelegrafPluginOutputInfluxDBV2.NameEnum.InfluxdbV2,
-    type: TelegrafPluginOutputInfluxDBV2.TypeEnum.Output,
-    config: {
-      urls: [`${window.location.origin}`],
-      token: '$INFLUX_TOKEN',
-      organization: name,
-      bucket,
-    },
-  }
-
-  const plugins = telegrafPlugins.reduce(
-    (acc, tp) => {
-      if (tp.configured === ConfigurationState.Configured) {
-        return [...acc, tp.plugin || createNewPlugin(tp)]
-      }
-
-      return acc
-    },
-    [influxDB2Out]
-  )
 
   if (telegrafConfigID) {
-    const telegraf = await client.telegrafConfigs.update(telegrafConfigID, {
-      name: telegrafConfigName,
-      description: telegrafConfigDescription,
-      plugins,
```

Comment:
By testing Load Data > Sources page by selecting a Telegraf Plugin, we still need plugins

Reply:
🤯 . @appletreeisyellow are we making a request to the `telegraf/plugin` endpoint or is it the regular `POST` to `/telegraf`?

Reply:
Here in the original code it is just a regular `POST`/`PUT` to `/telegraf`, but the influx API has the functionality to update the plugins. We might want to make a request to both `POST`/`PUT` `/telegraf` and `telegraf/plugin` for the new endpoint. 

Yes, we need the `plugins` for both the creation AND update step.

Seems like we need

Reply:
Not sure i follow, I don't see any addition stuff happening in the client API that updates the plugins besides just passing them along. Let's revert these changes for now, make the changes requested in other parts and deal with the post/put separately in a follow-up PR

Reply:
Sorry it was confusing. We were making a request to the regular `POST` to `/telegraf`.

Reply:
We are able to include the plugins in the request body of POST `/telegrafs` and PUT `/telegrafs/${params.telegrafID}` after a [fix](https://github.com/influxdata/openapi/pull/246) on OpenAPI. Thanks @glinton!!